### PR TITLE
Added a Comic, You're Welcome

### PR DESCRIPTION
**Pull request by dHelmgren**
## Description
I really don't know if this qualifies as an improvement?

### CODE REVIEW

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
 
 ![VCR](images/vcr.jpg)
 
+### What is VCR doing?
+VCR is essentially a recording device that interrupts the normal way they we send messages out to the internet. The first time a set of tests that use VCR run, the cassette takes note of the new URL:
```
Comment (tildeee) [inline 02-intermediate-ruby/api-testing-vcr.md]: Suggestion: the cassette takes note of the new ~URL~ request, based off of request method, URI, and body:

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
...
+
+![A VHS cassette watches a message leave HTTParty, taking note](images/apis/VCR_step_1.png "A VHS cassette watches a message leave HTTParty, taking note")
+
+When the API responds, the cassette takes note of the contents of the response as well:
+
+![A VHS cassette sees the response from the API before it gets back to HTTParty.](images/apis/VCR_step_2.png "A VHS cassette sees the response from the API before it gets back to HTTParty.")
+
+Whenever that URL is sent out in the future, VCR gobbles up the message before it goes to the API, taking the API out of the equation:
```
Comment (tildeee) [inline 02-intermediate-ruby/api-testing-vcr.md]: Suggestion: Whenever the same request (matching on request method, URI, and body) is sent out in the future, ...

**Comment by dHelmgren** [comment]
[Devin Added A Garbage Comic to VCR](https://trello.com/c/bBVYDJTY/1368-devin-added-a-garbage-comic-to-vcr)

**Comment by dHelmgren** [comment]
Merging this in! I did as dee said

**Pull request merged**